third_party/meerkat/Component/mmSH/android/server_runner_jni.cpp

```diff
@@ -169,7 +169,11 @@ jint Native_startServer(JNIEnv* env, jobject /* this */) {
 
   ServerRunner::ServerRunnerParams params;
   // TODO(yh106.jung): Read from configuration file
+#if defined(SERVICE_OFFLOADING)
```

Comment:
SERVICE_OFFLOADING flag is always enabled.
Please use a runtime flag.

Reply:
done